docs/sources/basics/timeseries.md

```diff
@@ -2,122 +2,7 @@
 title = "Time series"
 description = "Introduction to time series"
 keywords = ["grafana", "intro", "guide", "concepts", "timeseries"]
-aliases = ["/docs/grafana/latest/guides/timeseries", "/docs/grafana/latest/getting-started/timeseries"]
-weight = 500
+weight = 400
 +++
 
-# Introduction to time series
```

Comment:
Why was this content removed?

Reply:
I am not sure what happened. I do not think I touched this topic.

Reply:
You might not have meant to, but you definitely did. Maybe you can figure out where you did it and revert the changes.

Reply:
Ah, ok I see what happened. Will get this fixed.

Reply:
Fixed.